src/Protocol/Protocol_1_9.cpp

```diff
@@ -455,8 +455,13 @@ void cProtocol_1_9_0::SendEntityEquipment(const cEntity & a_Entity, short a_Slot
 
 	cPacketizer Pkt(*this, GetPacketId(sendEntityEquipment));  // Entity Equipment packet
 	Pkt.WriteVarInt32(a_Entity.GetUniqueID());
-	// Needs to be adjusted due to the insertion of offhand at slot 1
-	if (a_SlotNum > 0)
+	// See https://wiki.vg/Protocol#Entity_Equipment
+	// TODO: Enable player left hand. Currently only user by armor stand.
+	if (a_SlotNum == 5)  // Left hand
+	{
+		a_SlotNum = 1;
+	}
```

Comment:
If you aren't using this anymore then it should be removed.

Reply:
As said in the comment, armor stand use it
However it's not used by player's class

```
void cArmorStand::SetOffHandEquipedItem(const cItem & a_LeftHand)
{
	m_LeftHand = a_LeftHand;
	if (m_World != nullptr)
	{
		m_World->BroadcastEntityEquipment(*this, 5, a_LeftHand);  // See above (In SpawnOn) about the slot ID
	}
}
```
Moreover, for compatibility issues, left hand was added at the end and not at the ID 1 like wants the protocol since 1.9
This ways, previous versions keep equipment working

Reply:
Okay, so why have you disabled it in `cPlayer` then?

Reply:
I just add the line `// a_Client.SendEntityEquipment(*this, 5, m_Inventory.GetOffHandEquipedItem());  // Use number 5 for compatibility issues - Not activated since left hand is not now managed !`

And like said, I haven't enabled it because `GetOffHandEquipedItem` doesn't exist
I research again, and I now found that's because it's called `GetShieldSlot`

Then I will update the code to make the shield visible by everyone, but I should test it before commit

Reply:
Does this work with a 1.8 client? I think you might need to add 
```cpp
if (a_SlotNum == 5)
{
	return;
}
```

_Before_ the `cPacketizer` is constructed to avoid sending items to an invalid slot.

Reply:
I confirm that it's doesn't make a bad behavior on 1.8 except that the player can't see other's left hand's content.
I added your part of code as suggested to avoid a noise.